src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs

```diff
@@ -134,7 +134,7 @@ public CdnjsCatalog(CdnjsProvider provider)
         /// <returns></returns>
         public async Task<ILibrary> GetLibraryAsync(string libraryId, CancellationToken cancellationToken)
         {
-            (string name, string version) = LibraryNamingScheme.Instance.GetLibraryNameAndVersion(libraryId);
+            (string name, string version) = LibraryIdToNameAndVersionConverter.Instance.GetLibraryNameAndVersion(libraryId, this._provider.Id);
```

Comment:
Nit: do we use "this." prefix? Normally it's either "_" prefix or "this." prefix for member variables.